### ISSUE TITLE
Inflating Native Ad issue with context (UnsupportedOperationException)

### ISSUE THREAD

**Issue by sebastian-hoebarth** [AppLovin/AppLovin-MAX-SDK-Android#168]
After updating to 11.1.1 or 11.1.2 
I get the a UnsupportedOperationException when the Native ad tries to inflate the View with the context
```
Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 2: TypedValue{t=0x2/d=0x7f040104 a=-1}
        at android.content.res.TypedArray.getColorStateList(TypedArray.java:598)
        at androidx.cardview.widget.CardView.<init>(CardView.java:127)
        at com.google.android.material.card.MaterialCardView.<init>(MaterialCardView.java:118)
```

```
val binder: MaxNativeAdViewBinder =
                MaxNativeAdViewBinder.Builder(R.layout.card_row_native_max_rounded)
                    .setTitleTextViewId(R.id.primary)
                    .setBodyTextViewId(R.id.body)
                    .setAdvertiserTextViewId(R.id.secondary)
                    .setIconImageViewId(R.id.icon)
                    .setMediaContentViewGroupId(R.id.media_view)
                    .setCallToActionButtonId(R.id.cta)
                    .setOptionsContentViewGroupId(R.id.native_ad_daa_icon_image)
                    .build()
nativeAd = MaxNativeAdView(binder, context)
```

If I create an instance with "nativeAd = MaxNativeAdView(null, binder, context)" it works. The other constructor takes the application context and that is wrong, please fix this.

**Comment by maxakash** [comment]
Hey @sebastian-hoebarth instead of using 
```
MaxNativeAdViewBinder.Builder(R.layout.card_row_native_max_rounded)
```

Use 
```
MaxNativeAdViewBinder.Builder(layoutInflater.inflate(R.layout.card_row_native_max_rounded,null))
```

**Comment by talklittle** [comment]
Same problem on version 11.2.1. `MaxNativeAdView` constructor should not force the provided context into an application context; application context either crashes or breaks the styles of the provided layout.

The workarounds mentioned -- 1) alternate `MaxNativeAdView` constructor and 2) `MaxNativeAdViewBinder.Builder` taking a View -- do work, but should not be necessary.

**Comment by thomasmso** [comment]
This issue has been fixed in v11.3.0.
